config/prod/bsmn-reprocessing-s3-bucket.yaml

```diff
@@ -0,0 +1,43 @@
+# Provision a general S3 bucket for submitting reference tissue project data to NDA
+# Will copy data from nda-bsmn-scratch to this bucket to facilitate upload to NDA
+template_path: remote/s3-bucket.yaml
+stack_name: bsmn-reprocessing-s3-bucket
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "BSMN"
+  # The resource owner
+  OwnerEmail: "kenneth.daily@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) Synapse username (default ""), required if AllowWriteBucket=true
+  #  SynapseUserName: 'jsmith'
+  # (Optional) Allow accounts, groups, and users to access bucket. (default is no access).
+  #  GrantAccess:
```

Comment:
if you plan to use this bucket with synapse you'll need to grant synapse access to. 

[Optional] if you want to access this bucket with the aws cli you'll need to add your user ARN as well. 

```
  GrantAccess:
    - 'arn:aws:iam::325565585839:root'   # grant synapse access 
    - 'arn:aws:iam::563295687221:user/kenneth.daily@sagebase.org'
```

Reply:
Done